newrelic/resource_newrelic_dashboard.go

```diff
@@ -0,0 +1,295 @@
+package newrelic
+
+import (
+	"bytes"
+	"fmt"
+	"log"
+	"strconv"
+
+	"github.com/hashicorp/terraform/helper/hashcode"
+	"github.com/hashicorp/terraform/helper/schema"
+	newrelic "github.com/paultyng/go-newrelic/api"
+)
+
+func stringInList(values []string) schema.SchemaValidateFunc {
```

Comment:
You can use this method instead: https://github.com/hashicorp/terraform/blob/master/helper/validation/validation.go#L74